src/ProjectNeon/Assets/Data/Templates/CardAction.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CardAction<T> : MonoBehaviour where T : Target

```

Comment:
Where is this `T` generic used? I see no usages in the code.

Reply:
@SilasReinagel Actually I intended to add a generic type to this class so we could have `CardAction`s with composable defined scopes (aka `Target`) at design time. Unfortunately we can't do this way, so we'll use the `Scope`  notion to solve this.